src/js/review/SubmitButtons.jsx

```diff
@@ -45,8 +45,8 @@ export default function SubmitButtons(props) {
     submitMessage = (
       <div className="usa-alert usa-alert-error schemaform-failure-alert">
         <div className="usa-alert-body">
-          <p className="schemaform-warning-header"><strong>We’re sorry, there was an error connecting to Vets.gov.</strong></p>
-          <p>Please check your Internet connection and try again. If the problem persists, please contact the Vets.gov Help Desk.</p>
+          <p className="schemaform-warning-header"><strong>We’re sorry, there was an error connecting to the server.</strong></p>
```

Comment:
Does this change provide a way for Vets.gov to still use the original language? I don't think there's an option in the config for overwriting the default language here, as there is for the presubmit info.

Reply:
There are a lot of things in this file including styling, text , and error messages that someone might want to configure. There is even some logic, and someone might want to put the retry logic back for example. I think this whole file should end up in the starter app because there's just too much to try and configure via properties in `formConfig`.

Reply:
Good point, there should be an easy way to swap out the entire component if you want something different.